### DISCUSSION TITLE
Nested Slider and Breakpoints

### DISCUSSION THREAD

**Discussion by DirtyHerby** [nolimits4web/swiper#4096] [Q&A]
<img width="816" alt="Bildschirmfoto 2021-01-13 um 12 08 40" src="https://user-images.githubusercontent.com/12224262/104445838-ebd69d00-5599-11eb-8866-6178d4a1a2fe.png">
I use a nested slider as a horizontal full width gallery slider (slides per view: 1).
When the outer slider has three slides per view, the nested slider overtake this setting and shows three slides per view, too. I want it to be always one slide in nested slider. Example attached.

**Comment by vltansky** [comment]
A demo in Stackblitz / Codepen / codesandbox would be very useful to help you :)

**Reply by DirtyHerby** [reply]
Hi @vltansky,
thanks for your reply.
I made a codepen myself and came across the error. :D My code was different in one place. Therefore, it came to the erroneous view.
https://codepen.io/DirtyHerby/pen/mdrvGVm
